1 HW_Standard/HW_Exeggcute.yaml

```diff
@@ -19,4 +19,27 @@ shiny: True
 ivs: {hp: 31, atk: 31, def: 31, spA: 31, spD: 31, spe: 31}
 evs: {hp: 100, atk: 252, def: 140, spA: 252, spD: 252, spe: 0}
 nature: Serious
-hidden: false
\ No newline at end of file
+hidden: false
+---
+species: Exeggcute
```

Comment:
This should be a costume set, not standard

Reply:
I was thinking about that too, but from what I could tell Costume is all about baby Pokémon cosplaying as other baby Pokémon. Unsure if Exeggcute and Falinks fit that category.

Reply:
exeggcute does, and i Would consider "the mon being dressed up as must be lowest evo" to be a soft rule that can be bent for good concepts, not a strict requirement.

Reply:
Fair enough. I’ll update that and add in your previous suggestions in a bit.